cypress/component/integration/Dropdown.cy.tsx

```diff
@@ -0,0 +1,289 @@
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-dropdown');
+      });
+    });
+
+    describe('default xs dropdown', () => {
+      beforeEach(() => {
+        cy.mount(
+          <div className="theme-dark">
+            <Dropdown
+              size={'xs'}
+              options={[
+                {
+                  icon: (
+                    <MassaToken
+                      size={32}
+                      className="bg-c-default rounded-full"
+                    />
+                  ),
+                  item: 'account 1 name',
+                  onClick: () => console.log('option 1'),
+                },
+                {
+                  icon: <MassaLogo size={32} />,
+                  item: 'account 2 name with a biiiiig content',
+                },
+                { icon: <MassaLogo size={32} />, item: 'account 3 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 4 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 5 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 6 name' },
+              ]}
+            />
+          </div>,
+        );
+      });
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-xs-dropdown');
+      });
+    });
+
+    describe('default md dropdown', () => {
+      beforeEach(() => {
+        cy.mount(
+          <div className="theme-dark">
+            <Dropdown
+              size={'md'}
+              options={[
+                {
+                  icon: (
+                    <MassaToken
+                      size={32}
+                      className="bg-c-default rounded-full"
+                    />
+                  ),
+                  item: 'account 1 name',
+                  onClick: () => console.log('option 1'),
+                },
+                {
+                  icon: <MassaLogo size={32} />,
+                  item: 'account 2 name with a biiiiig content',
+                },
+                { icon: <MassaLogo size={32} />, item: 'account 3 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 4 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 5 name' },
+                { icon: <MassaLogo size={32} />, item: 'account 6 name' },
+              ]}
+            />
+            ,
+          </div>,
+        );
+      });
+
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+      });
+
+      it('should match snapshot', () => {
+        compareSnapshot(cy, 'default-md-dropdown');
+      });
+    });
+
+    describe('default dropdown flow', () => {
+      it('should render', () => {
+        cy.get('[data-testid="dropdown"]').should('exist');
+
+        it('should toggle when clicked', () => {
```

Comment:
Is there any reason to `should toggle` and other tests be nested under `should render` ones?

Reply:
no it's an auto complete bug